po/fr.po

```diff
@@ -17507,9 +17512,8 @@ msgid "Case insensitive searching is available on 'Account Full Name'."
 msgstr ""
 
 #: gnucash/gtkbuilder/dialog-find-account.glade:230
-#, fuzzy
 msgid "_Search"
-msgstr " Recherche "
+msgstr "Recherche"
```

Comment:
@ImagesLD2503 please read "_" (underline) in https://wiki.gnucash.org/wiki/Translation#Special_characters_and_other_tips